docs/docs/static-folder.md

```diff
@@ -17,25 +17,14 @@ You can create a folder named `static` at the root of your project. Every file y
 
 ### Referencing your static asset
 
-In order to reference assets from the `static` folder in your code, you'll need to [import the `withPrefix` helper function from `gatsby`](/docs/gatsby-link/#prefixed-paths-helper):
+You can reference assets from the `static` folder in your code without anything special required:
 
 ```js
-import { withPrefix } from 'gatsby'
-
 render() {
   // Note: this is an escape hatch and should be used sparingly!
   // Normally we recommend using `import` for getting asset URLs
   // as described in the “Adding Images, Fonts, and Files” page.
```

Comment:
I remember this page removed by this pull request? Should we add an link to an replacing page in normal text

Reply:
Which page?